pkg/kubelet/certificate/kubelet.go

```diff
@@ -61,6 +61,15 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 		},
 	)
 	legacyregistry.MustRegister(certificateExpiration)
+	var certificateRenewFailure = compbasemetrics.NewCounter(
```

Comment:
This assumes that this function will only be called once which is not obvious... but I guess that's how we do thinks. I'd probably prefer moving both of these to package level, but won't block this change.